bigchaindb/commands/utils.py

```diff
@@ -123,6 +123,55 @@ def input_on_stderr(prompt='', default=None, convert=None):
     return _convert(value, default, convert)
 
 
+def start_mongodb():
+    """Start mongo shell as a process and check if mongodb server is started
+    with replicaSet. If yes it checks if database is ready.
+    If yes it prints out the indexes in each of the collections of database
+
+    Raises:
+        :class:`~bigchaindb.common.exceptions.StartupError` if
+            mongo shell cannot be started.
+    """
+    import pymongo
+    import logging
+
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
+    host = str(bigchaindb.config['database']['host'])
+    port = str(bigchaindb.config['database']['port'])
+    hostport = host + ':' + port
+    dbname = bigchaindb.config['database']['name']
+    replname = bigchaindb.config['database']['replicaset']
+    proc = subprocess.Popen(['mongo', '--host', host, '--port', port],
```

Comment:
Pymongo driver is used for connections in the `start_mongodb` in subsequent checks.
mongo can be replaced if not needed.